source/includes/artifacts/_028-get-publish-artifact-metadata.md.erb

```diff
@@ -0,0 +1,40 @@
+## Get Publish Artifact Metadata
+
+This is a message that the plugin should, to allow users to configure global artifact stores from the Artifact Stores View in GoCD.
```

Comment:
`configure global artifact stores from the Artifact Stores View in GoCD` -> `configure external artifacts from the Job Config View in GoCD`